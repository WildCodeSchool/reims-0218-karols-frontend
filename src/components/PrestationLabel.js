import React from "react"

export default class PrestationLabel extends React.Component {
  render() {
    return (
      <div>
        <h2 className="mt-3 mb-4 display-4">
          Choix de votre pr√©paration Maquillage
        </h2>
      </div>
    )
  }
}
