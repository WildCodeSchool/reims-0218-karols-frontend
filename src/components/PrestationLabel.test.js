import React from "react"
import ReactDOM from "react-dom"
import renderer from "react-test-renderer"
import PrestationLabel from "./PrestationLabel"

it("renders without crashing", () => {
  const div = document.createElement("div")
  ReactDOM.render(<PrestationLabel />, div)
  ReactDOM.unmountComponentAtNode(div)
})

it("renders correctly", () => {
  const tree = renderer.create(<PrestationLabel />).toJSON()
  expect(tree).toMatchSnapshot()
})
