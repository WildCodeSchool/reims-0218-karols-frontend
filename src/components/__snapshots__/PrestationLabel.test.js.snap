// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <h2
    className="mt-3 mb-4 display-4"
  >
    Choix de votre pr√©paration Maquillage
  </h2>
</div>
`;
